### PR TITLE
[WIP] RegionFile fixes and improvements

### PR DESCRIPTION
**Pull request by astei**
This is still a work in progress, but in short:

* Resolves chunk-writing issues.
* Fixes chunk header reading.
* Removes chunk timestamps since they are not used.

### CODE REVIEW

**Review by Pr0methean — APPROVED** [review]

File: src/main/java/net/glowstone/io/anvil/RegionFile.java [hunk 1]
```diff
@@ -194,20 +191,14 @@ public RegionFile(File path) throws IOException {
...
-                }
+                sectorsUsed.set(startSector, startSector + numSectors + 1);
             } else if (offset != 0) {
                 GlowServer.logger.warning(
                         "Region \"" + path + "\": offsets[" + i + "] = " + offset + " -> "
                                 + startSector
                                 + "," + numSectors + " does not fit");
             }
         }
-        // read timestamps from timestamp table
```
Comment (mastercoms) [inline src/main/java/net/glowstone/io/anvil/RegionFile.java:207]: Although they are not used, they are part of the spec. Could you change usage of timestamps to be a configurable option?

**Review by mastercoms — CHANGES_REQUESTED** [review]

**Comment by astei** [comment]
I am still having issues with alignment, but the server is no longer crashing and persists changes to the world correctly. I will try to get some more time to finalize this soon.

**Comment by astei** [comment]
Won't be available for the next three days. Will resolve your concerns by then.

**Comment by astei** [comment]
Sorry for my late responses, college has kept me busy.

@mastercoms I readded timestamps.

**Comment by mastercoms** [comment]
Is this still WIP?

**Comment by astei** [comment]
There is still an unaligned error but worlds load and save changes correctly, and I'm not sure what's causing it. It might be macOS or something...

**Review by mastercoms — APPROVED** [review]

**Comment by mastercoms** [comment]
Ok, in favor of at least fixing some errors, I'll merge.

**Pull request merged**